### PR TITLE
Enable Firefox's native table editing

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: src/wymeditor/editor/gecko.js [hunk 1]
```diff
@@ -146,7 +146,8 @@ WYMeditor.WymClassGecko.prototype._designModeQuirks = function () {
     try {
         wym._doc.execCommand("styleWithCSS", '', false);
         wym._doc.execCommand("enableObjectResizing", false, false);
-        wym._doc.execCommand("enableInlineTableEditing", false, false);
```
Comment (mightyiam) [inline src/wymeditor/editor/gecko.js]: This enables the by-default Firefox's native table editing thingo, which looks like this:
screenshot_2015-11-21_20-06-32
It works quite alright, it seems, during some tinkering.

**Comment by mightyiam** [comment]
More discussion in #762.

**Comment by winhamwr** [comment]
:+1: 

Looks good! Does Chrome have an equivalent?

**Comment by mightyiam** [comment]
@winhamwr Chrome doesn't seem to have such a feature. Doesn't surprise me at all.
http://stackoverflow.com/questions/25372336/enableobjectresizing-and-enableinlinetableediting-in-webkit

**Comment by mightyiam** [comment]
Thank you!

File: src/wymeditor/editor/gecko.js (hunk 1 above)
Comment (Geetha2111) [reply src/wymeditor/editor/gecko.js:149]: It doesn't support other browsers like chrome or safari. Is there any other alternative for this.?

Comment (mightyiam) [reply src/wymeditor/editor/gecko.js:149]: Brah, WYMeditor is such an old editor. Why don't you use some more modern alternative?

**Pull request merged**